netbox/dcim/migrations/0200_populate_mac_addresses.py

```diff
@@ -51,3 +53,42 @@ class Migration(migrations.Migration):
             name='mac_address',
         ),
     ]
+
+
+def oc_interface_primary_mac_address(objectchange, reverting):
```

Comment:
Might be worth leaving a note for future maintainers to look at the migrator for VMInterface as well if making changes here.